README.md

```diff
@@ -68,3 +69,26 @@ Add the following to `layout.html` that all of your pages extend, or to `outerLa
   {% endif %}
 {% endblock %}
 ```
+
+If you would like to configure additional fields to allow an editor to add a Google Analytics tracking ID and a Google site verification ID you can do so by setting `seoGoogleFields: true` in `apostrophe-global` in your project.
+
+Add the following to `layout.html` that all of your pages extend, or to `outerLayout.html` if you have one in `apostrophe-templates/views/` to add configurable Google Analytics and Google site verification.
```

Comment:
How about a file they can `{% include %}`? That would be nice for all of the insertions in the layout. They can always copy it in and modify it, or override one or another one at project level, if it's not quite what they want.

Reply:
Moved this into a view per your suggestion b0ff2f0